packages/stems/src/styles/types.ts

```diff
@@ -0,0 +1,20 @@
+import { CSSProperties } from 'react'
+
+/** Type override to allow usage of custom properties in the
+ * `style' prop. We're using this instead of augmenting the csstype module
+ * because that currently breaks types for some third-party modules. Note that
```

Comment:
👀 what does it break?

Reply:
The normal strategy is to [augment the csstype module](https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors) to allow custom properties. That type then gets inherited by any third-party module that derives from `CSS.Properties`. React's `CSSProperties` is included.

When I did that, it broke [this call to `useTransition`](https://github.com/AudiusProject/audius-client/blob/main/packages/stems/src/components/Modal/Modal.tsx#L194). The generic used for that hook expects the values of `from`, `to`, etc to be extensions of `CSSProperties`. However, it believed that `transform` wasn't a valid property. I explored for a while and couldn't find a module augmentation solution that worked without breaking some types that depended on `CSSProperties`. It seemed to me like the types weren't being merged? But even when I tried copying the `React.CSSProperties` definition and extending it to include custom properties, things still didn't work.

So! Deep deep type resolution problem that didn't feel worth the time down the rabbit hole.

Reply:
Interesting, surely there is a way to augment the type and make everything happy. But good call on manually defining the type for now, maybe we can figure it out in the future